test/test.js

```diff
@@ -1,35 +1,461 @@
+                            observable: true,
+                            readOnly: false,
+                            writeOnly: false,
+                            forms: [
+                                {
+                                    href:
+                                        "http://localhost:8081/TestServient/properties/display",
+                                    contentType: "application/json",
+                                    op: ["readproperty", "writeproperty"],
+                                },
+                                {
+                                    href:
+                                        "http://localhost:8081/TestServient/properties/display/observable",
+                                    contentType: "application/json",
+                                    op: ["observeproperty"],
+                                    subprotocol: "longpoll",
+                                },
+                                {
+                                    href:
+                                        "coap://localhost:8082/TestServient/pr/display",
+                                    contentType: "application/json",
+                                    op: ["readproperty", "writeproperty"],
+                                },
+                            ],
+                        },
+                        counter: {
+                            type: "number",
+                            writable: true,
+                            observable: true,
+                            readOnly: false,
+                            writeOnly: false,
+                            forms: [
+                                {
+                                    href:
+                                        "http://localhost:8081/TestServient/properties/counter",
+                                    contentType: "application/json",
+                                    op: ["readproperty", "writeproperty"],
+                                },
+                                {
+                                    href:
+                                        "http://localhost:8081/TestServient/properties/counter/observable",
+                                    contentType: "application/json",
+                                    op: ["observeproperty"],
+                                    subprotocol: "longpoll",
+                                },
+                                {
+                                    href:
+                                        "coap://localhost:8082/TestServient/pr/counter",
+                                    contentType: "application/json",
+                                    op: ["readproperty", "writeproperty"],
+                                },
+                            ],
+                        },
+                        temperature: {
+                            type: "number",
+                            writable: false,
```

Comment:
writable could be removed

Reply:
writable statements are removed in newest commit.